developer/architecture/schemas.md

```diff
@@ -44,7 +44,7 @@ registerSchema("MyNewSchema", MyNewSchema);
 All registered schemas can be retrieved by importing `getSchemas`, and then using [`getSchemas()`](http://api.docs.reactioncommerce.com/module-collections.html#~getSchemas).
```

Comment:
This line can be removed.  `getSchemas` was removed because it was just a function with no logic that returned the `Schemas` object.  It required an import and then an additional unnecessary function call just to get the `Schemas` object when you could just simply import `Schemas` directly in one shot.